src/utils/RacingCarUtils.js

```diff
@@ -0,0 +1,53 @@
+import {Console, Random} from '@woowacourse/mission-utils';
+import RacingCar from "../RacingCar.js";
+import ValidationUtils from "./ValidationUtils.js";
+import ErrorCode from "../datas/ErrorCode.js";
+
+class RacingCarUtils {
+    static findWinners(carList) {
+        if (carList.length <= 0) throw new Error(ErrorCode.CAR_NOT_ENOUGH)
+
+        carList.sort((a, b) => b.getDistance() - a.getDistance())
+
+        const maxDist = carList[0].getDistance();
+        return carList.filter(car => car.getDistance() === maxDist)
+            .map(car => car.name);
+    }
+
+    static createCarListByPlayerList(players) {
+        if (ValidationUtils.checkDuplicate(players)) {
+            throw new Error(ErrorCode.CAR_NAME_DUPLICATE)
+        }
+        return players.reduce((cars, player) => {
+            cars.push(new RacingCar(player));
+            return cars;
+        }, []);
```

Comment:
```suggestion
        return players.reduce((cars, player) => [...cars, new RacingCar(player)], []);
```
스프레드 문법으로 간략하게 작성하는건 어떨까요??

Reply:
아직 js문법에 많이 익숙하지 않은게, 이번 프리코스를 하면서 많이 느껴지네요 ㅠㅠ
감사합니다!